README.md

```diff
@@ -14,20 +14,23 @@ This repository contains the Puppet Module for installing and configuring WSO2 I
 ## How to Contribute
 Follow the steps mentioned in the [wiki](https://github.com/wso2/puppet-base/wiki) to setup a development environment and update/create new puppet modules.
 
+## Dependencies
```

Comment:
Shall we put the same as in IS-ANALYTICS, i.e. WSO2 IS 5.4.0 puppet module is compatible and tested with" ? this will be much clear than mentioning the dependencies...